test/data/doubles_multiline_string.py

```diff
@@ -1,7 +1,7 @@
-""" This "should"
+s = """ This "should"
```

Comment:
When there is no assignment, it's considered a docstring. When there is an assigment, it's not considered a docstring and I believe that's the correct behavior. I just wanted all existing test cases to not contain docstrings, i.e. make them test existing behavior and add new tests for docstrings. Is there something I'm missing?